### PR TITLE
try to capture comments/resolution to AD comments

### PR DESCRIPTION
**Pull request by mcr**
attempt to provide forward reference as requested on Monday

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -572,6 +572,9 @@ As explained in {{RFC4949}}, Section 2.6 when this document says "Compare:", the
 
 This document uses the following terms.
 
+As explained in {{overview}}, artifacts are defined by which role elements produce them or consume them.
+Other role elements may forward, pass-through, cache or repeat artifacts.
```
Comment (thomas-fossati) [inline draft-ietf-rats-architecture.md:576]: what does "repeat" mean?  is it "make identical copies"?

Comment (dthaler) [inline draft-ietf-rats-architecture.md:576]: To Thomas's point, maybe:
```suggestion
As explained in {{overview}}, artifacts are defined by which roles produce them or consume them.
In some protocol instantiations, other roles may cache and forward artifacts as opaque data, to one or entities implementing the consuming role.
```

**Review by dthaler — APPROVED** [review]

**Comment by dthaler** [comment]
This looks good to me, although I wonder whether the sentence would make more sense at the end of the "Implementation Considerations" section (a couple lines earlier in the document).  I am ok either way.

**Comment by mcr** [comment]
> This looks good to me, although I wonder whether the sentence would make more sense at the end of the "Implementation Considerations" section (a couple lines earlier in the document). I am ok either way.

Works for me. That would put that into section 1.

**Review by thomas-fossati — APPROVED** [review]
LGTM

**Review by nedmsmith — COMMENTED** [review]
"artifacts are defined by which roles produce them or consume them" - This sentence structure seems awkward. Is "artifacts are defined by the roles that produce or consume them" better?

**Pull request merged**